### PR TITLE
feat(participant): participant & participant list components

### PR DESCRIPTION
**Pull request by stevem63**
Add participant and participant list components. New hook was created to get a list of participants based on a location identifier, so that it can be used for meetings or spaces with any type of unique identifier.

https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-97490
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-97491

### CODE REVIEW

File: src/components/hooks/useParticipants.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    const onError = (error) => {
+      setParticipants({
+        members: [],
+      });
+
+      // eslint-disable-next-line no-console
+      console.error(error.message);
+    };
+    const onMembers = (data) => {
+      setParticipants(data);
```
Comment (lalli-flores) [inline src/components/hooks/useParticipants.js:25]:
```suggestion
      setParticipants(data.members);
```

File: src/data/members.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "location_1": [
```
Comment (lalli-flores) [inline src/data/members.json:2]: The data model should look like:
```js
"default_membership": {
  "ID": "default",
  "destinationID" : ""
  "members": [
    "personID": "",
    "personID": ""
       ....
  ]
}
```
Where `ID` is the ID of the membership, `destinationID` is the ID of either the meeting or room, and then the `members` is an array of `personID`s

File: src/adapters/MembershipJSONAdapter.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  /**
+   * @param {string} ID A unique identifier for a meeting/space location
+   * @returns {Observable.<Membership>}
+   */
+  getMembers(ID) {
+    return Observable.create((observer) => {
+      if (this.datasource[ID]) {
+        observer.next(this.datasource[ID]);
+      } else {
+        observer.error(new Error(`Could not find members with location identifier "${ID}"`));
```
Comment (lalli-flores) [inline src/adapters/MembershipJSONAdapter.js:48]:
```suggestion
        observer.error(new Error(`Could not find members for destination "${destinationID}"`));
```

File: src/components/WebexParticipantList/WebexParticipantList.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+ * @returns {object} JSX of the component
+ */
+export default function WebexParticipantList({location}) {
+  const participants = useParticipants(location);
+
+  const participantList = map(participants, (participant) => (
+    <WebexParticipant personID={participant.ID} displayStatus={participant.displayStatus} />
+  ));
+
+  return <List>{participantList}</List>;
```
[the remark this answers is not in the document]
Comment (stevem63) [reply src/components/WebexParticipantList/WebexParticipantList.js:20]: Yep, updated

**Pull request merged**